egs2/TEMPLATE/asr1/asr_transcript.sh

```diff
@@ -0,0 +1,1626 @@
+#!/usr/bin/env bash
```

Comment:
My major concern with the current framework is whether we would want to hold this recipe as in `asr1` as it is not fully aligned with the asr definition (more likely to be aligned with slu specifically). I'm not sure, but maybe we could make a standalone SLU framework with this PR?

Reply:
I agree with @ftshijt.
I think it's time to make an SLU specific template.
`egs2/TEMPLATE/slu1/asr_transcript.sh` ???